src/stylesheet/00_settings/_dimen.scss

```diff
@@ -0,0 +1,81 @@
+$login: (
```

Comment:
What is the purpose of this file? Does it just define dimension variables for certain components?
If so why not just define them in the component stylesheets as these should never be accessed out the scope of the login component?